src/web/app/components/simple-modal/simple-modal.component.spec.ts

```diff
@@ -11,9 +12,9 @@ describe('SimpleModalComponent', () => {
     TestBed.configureTestingModule({
       declarations: [SimpleModalComponent],
       providers: [NgbActiveModal],
-      imports: [ProgressBarModule],
+      imports: [ProgressBarModule, AjaxLoadingModule],
     })
-    .compileComponents();
+        .compileComponents();
```

Comment:
Any reason for this indentation?

Reply:
This is me over-linting, I reverted this change.